source/draft_errors.rst

```diff
@@ -165,23 +155,23 @@ server is running.
     {
         "status": 401,
         "error": "Authentication failure",
-        "text": "Invalid password" / "Bad oAuth token" / etc
+        "text": "Invalid username or password" / "Bad oAuth token" / etc
     }
 
 ----------------------
 
-8. Slug already exists
+7. Slug Already Exists
 ----------------------
 
-This error is returned when a new object is being created but the slugs passed
-in contain a slug that already exists.
+This error is returned when a new object is being created but the slugs provided
+contain a slug that already exists.
 
 .. code-block:: javascript
 
     {
         status: 409,
```

Comment:
In the code-block examples above, each fieldname is encased in quotes. Starting with Error 7, that consistency is broken.